treeshr/TreeSerializeNci.c

```diff
@@ -29,65 +29,65 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 void TreeSerializeNciOut(const NCI *in, char *out)
 {
-  char *ptr = out;
```

Comment:
I like the intention but the change does not handle endianess (handled by put/getint32/64)